srcpkgs/ghcup/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'ghcup'
+pkgname=ghcup
+version=0.1.17.4
+revision=1
+wrksrc="ghcup-hs-v$version"
+build_style='haskell-stack'
+make_build_args='--flag ghcup:tui --flag ghcup:internal-downloader'
+hostmakedepends='tar'
+makedepends='zlib-devel ncurses-libtinfo-devel openssl-devel'
+depends='zlib ncurses-libtinfo-libs openssl'
```

Comment:
`curl` and `wget` are optional runtime dependencies.

Additionally, you need at least these **runtime dependencies** to install GHC via ghcup: `gcc gmp make ncurses ncurses-libtinfo-libs perl`

Reply:
the runtime dependencies have been added, and optional curl/wget dependencies are mentioned in INSTALL.msg.